src/css/core.scss

```
@@ -322,8 +322,10 @@ nav {
 		height: 27px;
 	}
 
-	&:hover > ul {
+	&:hover > ul,
+	&:focus > ul {
 		display: flex;
+		opacity: 100;
```

Per the above suggestion, re-enable pointer events on subheading elements once they become visible.

```suggestion
		pointer-events: all;
		opacity: 100;
```